docs/stable/2-account.md

```diff
@@ -39,6 +39,13 @@ This specification explains what Status account is, and how a node establishes t
         - [Identicon](#identicon)
         - [3 word pseudonym / Whisper/Waku key fingerprint](#3-word-pseudonym--whisperwaku-key-fingerprint)
         - [ENS name](#ens-name)
+- [Public Key Compression](#public-key-compression)
```

Comment:
I'm missing some rationale for this feature. I know the gist of it but it'd be good to have it documented. Where exactly is it used etc?

Reply:
I've added a new subsection detailing where exactly this functionality would be used and why this functionality would be used. https://github.com/status-im/specs/pull/137/commits/d4cc96c42baf784c5890b4fdd4ebd5e4badfcb3e